apps/website/src/content/docs/anchor.mdx

```diff
@@ -57,6 +57,14 @@ If you do ultimately decide based on your research that it is beneficial to open
 
 **Note**: Not all external links should open in a new tab. The `isExternal` prop is not necessary for external links that open in the same tab.
 
+### Underlines
+
+By default, the anchor is only underlined when hovered, except when using a high-contrast theme. To explicitly make the anchor underlined in all themes, use the `underline` prop.
```

Comment:
non-blocking: Maybe this part can also mention the accessibility benefits of having underlines on anchors.

Reply:
[`bd36389`](https://github.com/iTwin/iTwinUI/pull/1864/commits/bd3638915f2fd2e610d8f0c53e6f371a0d379392) added link for now.